mesh/submesh/psubmesh.hpp

```diff
@@ -342,9 +392,16 @@ class ParSubMesh : public ParMesh
    void BuildSharedFacesMapping(const int nstrias, const Array<int>& rht,
                                 const int nsquads, const Array<int>& rhq);
 
+
+   std::unordered_map<int, int>
+   FindGhostBoundaryElementAttributes() const;
+
    /// The parent Mesh
    const ParMesh &parent_;
 
+   /// Optional nonconformal submesh. Managed via ncmesh pointer in base class.
```

Comment:
`ncmesh` -> `pncmesh`?

Reply:
They both point to the same so shouldn't make a difference, but will change.